gcc/rust/resolve/rust-early-name-resolver-2.0.cc

```diff
@@ -135,13 +135,28 @@ Early::visit (AST::MacroInvocation &invoc)
   // if the definition still does not have a value, then it's an error
   if (!definition.has_value ())
     {
-      rust_error_at (invoc.get_locus (), ErrorCode::E0433,
-		     "could not resolve macro invocation");
+      collect_error ([&] () {
+	rust_error_at (invoc.get_locus (), ErrorCode::E0433,
+		       "could not resolve macro invocation");
+      });
```

Comment:
so this would become something like
```suggestion
      add_error (Error (invoc.get_locus (), ErrorCode::E0433,
		       "could not resolve macro invocation"));
```

Reply:
Fixed in fdccd48aff5.